CMakeLists.txt

```diff
@@ -19,25 +19,29 @@ if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
 endif()
 
 find_package(Franka 0.7.0 REQUIRED)
-find_package(Eigen3 REQUIRED)
 find_package(cppzmq REQUIRED)
+find_library(protobuf REQUIRED)
+find_library(clproto REQUIRED)
+find_library(state_representation REQUIRED)
+find_package(Eigen3 REQUIRED)
 
 include_directories(
  	include
  	${Eigen3_INCLUDE_DIRS}
+	/usr/local/include/eigen3/
```

Comment:
For some reason my Docker installation couldn't resolve the `#include <Eigen/Core>` lines, since they are installed to `/usr/local/include/eigen3/Eigen` - and `${Eigen3_INCLUDE_DIRS}` apparently didn't populate the search path. I can take it out again and instead revise the include to use `#include <eigen3/Eigen/Core>`